backend/apis/leads/add-leads.api.js

```diff
@@ -10,7 +10,7 @@ const {
 } = require("../utils/validation-utils");
 
 app.post("/api/leads", (req, res) => {
-  const leads = req.body.data;
```

Comment:
The api documentation does not call for a `data` property to be in the request body